From a4b11bf92b986c1612843342aac9f6aa8bedb357 Mon Sep 17 00:00:00 2001
From: jkreierh <jkreierh@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 31 Aug 2017 11:42:06 +0000
Subject: [PATCH 0875/1077] MANTIS 47240 Mantis 47240: Config service meta data
 access - Add semantic type ST_UNKNOWN, to indicate legacy data

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1253 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/cfgmgr.idl | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/cfgmgr.idl b/SelfServiceCommon/Massai/idl/Massai/cfgmgr.idl
index 6bc607e6..a57a20de 100644
--- a/SelfServiceCommon/Massai/idl/Massai/cfgmgr.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/cfgmgr.idl
@@ -51,11 +51,12 @@ module Config
     /** The type codes here might help to validate an input */
     enum semanticType
     {
-        ST_NONE,                        /**< No special type */
-        ST_PASSWORD,                    /**< Password */
-        ST_URL,                         /**< Generic URL */
-        ST_HOSTNAME,                    /**< Hostname or IP Address*/
-        ST_IP_ADDRESS                   /**< IP Address */
+        ST_UNKNOWN,             /**< Not assigned (compatibility) */
+        ST_NONE,                /**< No special type */
+        ST_PASSWORD,            /**< Password */
+        ST_URL,                 /**< Generic URL */
+        ST_HOSTNAME,            /**< Hostname or IP Address*/
+        ST_IP_ADDRESS           /**< IP Address */
     };
 
 
-- 
2.41.0.windows.1

